recipes/patsy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.4.1" %}
+{% set gh_org = "pydata" %}
+{% set gh_repo = "patsy" %}
+{% set pkg_name = "patsy" %}
+
+
+package:
+  name: {{ pkg_name }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkg_name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ pkg_name[0] }}/{{ pkg_name }}/{{ pkg_name }}-{{ version }}.zip
+  sha256: dc1cc280045b0e6e50c04706fd1e26d2a00ea400aa112f88e8142f88b0b7d3d4
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.4.0
+    - scipy
+    - six
+
+test:
+  imports:
+    - patsy
+
+about:
+  home: https://github.com/{{ gh_org }}/{{ gh_repo }}
+  license: BSD-2-Clause
+  license_family: BSD
+  summary: Describing statistical models in Python using symbolic formulas
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - wdm0006
```

Comment:
Sure, I'm in... @wdm0006 was perhaps an oversight, as I/he was mostly interested in showing him how `category_encoders` could be packaged for conda... the patsy recipe was a sideeffect!